corehq/apps/app_manager/templates/app_manager/partials/releases_deploy_modal.html

```diff
@@ -47,6 +47,10 @@ <h4 class="modal-title">{% trans "Deploying version" %} <span data-bind="text: v
                                                 </a>
                                                 {% trans "to your Android device." %}
                                             </p>
+                                            <label>{% trans "Build Profile" %}</label>
+                                            {% if build_profile_access %}
+                                            <select data-bind="optstr: build_profiles(), value: profile" class="form-control"></select>
```

Comment:
Do you know if this is smart enough to display them in like alphabetical order or something?